src/main/resources/com/devonfw/ide/sonarqube/common/rules/devon4j/devon4j.xml

```diff
@@ -126,30 +126,6 @@
         <priority>MINOR</priority>
         <parameters/>
     </rule>
-    <rule>
-        <repositoryKey>checkstyle</repositoryKey>
-        <key>com.puppycrawl.tools.checkstyle.checks.annotation.AnnotationLocationCheck</key>
```

Comment:
OK, so this rule has been removed to fix #117.
For me as a reviewer it always helps if we have a rationale to understand why we drew this conclusion.
I can not see that this rule was removed from checkstyle plugin for sonarqube:
https://github.com/checkstyle/sonar-checkstyle/blob/master/src/main/resources/org/sonar/l10n/checkstyle.properties#L523

Reply:
It should definitely be available, but it seems that with versions 8.35 and 8.37, SonarQube sees this check as a 'rule template' and not as a rule by itself. This is because in the default config of this rule, `cardinality` is set to `multiple`. I am working on a fix right now that sets this back to single

Reply:
As discussed with @lmarniazman this rule was identified from the stacktrace and the error is saying that this rule is only a template and needs to be parameterized to be instanciated. If we want to keep it in our quality profile, we have to parameterize it somehow. We can do that for the next release if we think we still need this rule.